From ae6c568fecbef8b2aa72a89d6afd4aabcec10a05 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 28 Jul 2023 14:24:04 +0100
Subject: [PATCH] curl: Rename unload_config, unload_pool -> config_unload,
 pool_unload

For consistency with scripts_unload.

There is still a function called load_pool, but that inconsistency
will be removed in a follow-on commit.

Reviewed-by: Eric Blake <eblake@redhat.com>
(cherry picked from commit f7bea74f670ba303c7ea60713cbbf301fab865fd)
---
 plugins/curl/config.c   | 2 +-
 plugins/curl/curl.c     | 4 ++--
 plugins/curl/curldefs.h | 4 ++--
 plugins/curl/pool.c     | 2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/plugins/curl/config.c b/plugins/curl/config.c
index d68c92ba..276c79d5 100644
--- a/plugins/curl/config.c
+++ b/plugins/curl/config.c
@@ -103,7 +103,7 @@ NBDKIT_DLL_PUBLIC int curl_debug_verbose = 0;
 NBDKIT_DLL_PUBLIC int curl_debug_verbose_ids = 0;
 
 void
-unload_config (void)
+config_unload (void)
 {
   free (cookie);
   if (headers)
diff --git a/plugins/curl/curl.c b/plugins/curl/curl.c
index 99a7e00b..72971093 100644
--- a/plugins/curl/curl.c
+++ b/plugins/curl/curl.c
@@ -74,9 +74,9 @@ curl_load (void)
 static void
 curl_unload (void)
 {
-  unload_config ();
+  config_unload ();
   scripts_unload ();
-  unload_pool ();
+  pool_unload ();
   display_times ();
   curl_global_cleanup ();
 }
diff --git a/plugins/curl/curldefs.h b/plugins/curl/curldefs.h
index 022e8c60..cab4a6b1 100644
--- a/plugins/curl/curldefs.h
+++ b/plugins/curl/curldefs.h
@@ -119,13 +119,13 @@ struct curl_handle {
 extern int curl_config (const char *key, const char *value);
 extern int curl_config_complete (void);
 extern const char *curl_config_help;
-extern void unload_config (void);
+extern void config_unload (void);
 extern struct curl_handle *allocate_handle (void);
 extern void free_handle (struct curl_handle *);
 
 /* pool.c */
 extern void load_pool (void);
-extern void unload_pool (void);
+extern void pool_unload (void);
 extern struct curl_handle *get_handle (void);
 extern void put_handle (struct curl_handle *ch);
 
diff --git a/plugins/curl/pool.c b/plugins/curl/pool.c
index 91e56f07..50a623a4 100644
--- a/plugins/curl/pool.c
+++ b/plugins/curl/pool.c
@@ -86,7 +86,7 @@ load_pool (void)
 
 /* Close and free all handles in the pool. */
 void
-unload_pool (void)
+pool_unload (void)
 {
   size_t i;
 
-- 
2.39.3

